mdcopy/mdcopy.go

```diff
@@ -0,0 +1,36 @@
+package mdcopy
```

Comment:
Rename this to be more specific such as responseheader

Reply:
done